android/app/build.gradle

```diff
@@ -155,7 +159,6 @@ repositories {
 
 
 dependencies {
-    compile project(':react-native-notifications')
```

Comment:
Will removing this be a problem?

Reply:
No,

It's already get linked in https://github.com/zulip/zulip-mobile/blob/master/android/app/build.gradle#L171

if we don't remove this line, then `react-native-notification` is linked twice and thus causing this error.

```
* What went wrong:
Execution failed for task ':app:processDebugResources'.
> Error: more than one library with package name 'com.wix.reactnativenotifications'
```

Reply:
Cool. The rn-link is causing this issues. I'll be more careful next time I run it.